test/ring/middleware/test/json.clj

```diff
@@ -337,6 +338,25 @@
       (is (= (get-in response [:headers "Content-Type"]) "application/json; some-param=some-value"))
       (is (= (:body response) "{\"foo\":\"bar\"}")))))
 
+(defn StreamableResponseBody->String [body]
```

Comment:
Thanks for the update. Can you change this function definition to `(defn- streamable->string`